tools/test.py

```diff
@@ -1636,6 +1637,11 @@ def Main():
   if has_inspector.stdout.rstrip() == "0":
       context.v8_enable_inspector = False
 
+  has_crypto = Execute([vm,
+      "-p", "process.versions.openssl"], context)
+  if has_crypto.stdout.rstrip() == "undefined":
+      context.node_has_crypto = False
```

Comment:
Style nit: two space indent and single quotes. (I realize the code above does it too but style deviations shouldn't be allowed to propagate.)

Reply:
Thanks, I'll update both.